src/autoadvance.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function autoadvance(player, timeout) {
+  var resetadvance = function resetadvance() {
+    if (player.playlist._timeoutId) {
+      window.clearTimeout(player.playlist._timeoutId);
+    }
+
+    if (player.playlist._ontimeout) {
+      player.off('ended', player.playlist._ontimeout);
+    }
+
+    player.playlist._timeoutId = null;
+    player.playlist._ontimeout = null;
+  };
+
+  // we are want to cancel the auto advance or accidentally called it with a bogus value
```

Comment:
Yes. We are [want](http://www.merriam-webster.com/thesaurus/want%5Bnoun%5D) to cancel the auto advance.